src/Context/ServerOsContext.php

```diff
@@ -186,7 +186,7 @@ public function setKernelVersion(string $kernelVersion): void
     private function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
-            'name' => php_uname('s'),
+            'name' => PHP_OS,
```

Comment:
Does it makes sense to change this instead of being coherent with the rest of the values using the `php_uname` function?

Reply:
Yea indeed using a constant is better than calling a runtime function, I'm not sure it was worth the change since we have anyway other calls `php_uname` and so removing one imho won't affect the performance. Either ways are fine by the way